frontend/src/assets/sass/outcome-report.scss

```diff
@@ -478,3 +478,16 @@
     margin-top: 16px;
   }
 }
+
+.outcome-authroization-unauthroized-site-prefix {
+  background-color: transparent;
+  border-right: none;
+  color: $gray-500;
+  border: 1px solid hsl(0, 0%, 80%);
+  padding-right: 0.1875rem;
+}
+
+.outcome-authroization-unauthroized-site-input {
+  border-left: none !important;
+  padding-left: 0.1875rem;
+}
```

Comment:
Are you able to style the input properly?

![image](https://github.com/user-attachments/assets/e74306a9-b0ba-48f1-ab54-15238b90ea98)

https://www.30secondsofcode.org/css/s/input-with-prefix/ has an example

Reply:
As discussed with UX will leave this as is for now.